src/TRUNAJOD/entity_grid.py

```diff
@@ -63,7 +63,7 @@ class EntityGrid(object):
     module. It only supports 2-transitions entity grid.
     """
 
-    def __init__(self, doc):
+    def __init__(self, doc, model="spacy"):
```

Comment:
I think to make easier further improvements to internals, we might create an extra module with the constants, something like:

```python
from enum import Enum


class SupportedModels(str, Enum):
    SPACY = "spacy"
    STANZA = "stanza"
```

Then in all instances we could use the enum. For example, the constructor arguments could be:

```python
def __init__(self, doc, model_name="spacy"):
    # The following might do error checking on model name, will raise if not in Enum
    model = SupportedModels(model_name)

    # Checks could be done as
    if model == SupportedModels.SPACY:

```

In that manner, all instances could be replaced by something like `model=SupportedModels`

Reply:
alright, I'll look into it